docker/docker-compose.yaml

```diff
@@ -44,33 +41,35 @@ services:
       - internal
     ports:
       - 5000:5000
-    links:
-      - conductor-server
     stdin_open: true
 
   redis:
     image: redis:6.2.3-alpine
     volumes:
       - ./redis.conf:/usr/local/etc/redis/redis.conf
     networks:
-      - internal
+      internal:
+        aliases:
+          - rs
     ports:
       - 6379:6379
     healthcheck:
       test: [ "CMD", "redis-cli","ping" ]
 
   elasticsearch:
-    image: elasticsearch:6.8.15
+    image: jazzdev/elasticsearch:6.8.15-arm
     container_name: elasticsearch
     environment:
-      - "ES_JAVA_OPTS=-Xms512m -Xmx1024m"
+      - "ES_JAVA_OPTS=-Xms1024m -Xmx1024m"
```

Comment:
elasticsearch outputs a warning if these aren't the same value.
Could set them both to 512 and maybe run the VM with only 4GB instead of 5GB.